x-pack/plugins/observability_solution/inventory/public/plugin.ts

```diff
@@ -51,6 +52,10 @@ export class InventoryPlugin
     const inventoryAPIClient = createCallInventoryAPI(coreSetup);
     this.telemetry.setup({ analytics: coreSetup.analytics });
 
+    const inventoryLocator = pluginsSetup.share.url.locators.create(
```

Comment:
What do you think of creating and exporting the locator [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_shared/public/plugin.ts#L138-L139) alongside with other solutions?

Reply:
I see you added the locator in the shared plugin, can we remove it from here?

Reply:
Oh, right! I completely forgot about that, thank you!